AoE2.netDesktopTests/PlotEx/PlayerRateFormsPlotTests.cs

```diff
@@ -86,8 +86,8 @@ public void PlotTest()
     {
         // Arrange
         var testClass = new PlayerRateFormsPlot(new FormsPlot(), leaderboardColor, 16);
-        var datetimeMaxX = new DateTime(1970, 01, 01, 9, 0, 0);
-        var datetimeMinX = new DateTime(1970, 01, 01, 9, 0, 0);

```

Comment:
You know, this test does not pass outside the UTC + 9 area.
Therefore, the expected value should be converted to the local date-time to pass the test in any time zone.
Fixed it on the following commit.

https://github.com/teshiba/AoE2.netDesktop/commit/f4f88dd7a9a10087c31260b10812b650e7d2b704?diff=unified#r77572087